rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -161,35 +161,33 @@ def rocalRun(self):
         """ Run the pipeline using rocalRun call
         """
         status = b.rocalRun(self._handle)
```

Comment:
if(status != types.OK):
            print("Rocal Run failed")

            Why have we removed the error check statement ? We might need to add this back

Reply:
I'm using this function for checking if the loader ran out of images so this print statement is getting printed at the end of each epoch so I removed this print statement

Reply:
Are you using build() call to check if the loader ran out of images?

Please add this back for error handling

Reply:
I use rocal_run function for checking stopIteration so after the loader runs out of images, it prints this which is not needed right? Else at every epoch end for all trainings, it prints this message

Reply:
@swetha097 Has this been checked?
Were we printing this at the end of every epoch before?

Reply:
Previously we were using b.isEmpty() to raise stop iteration which doesn't have the print statement

Reply:
I think we can remove this . Prev b.isEmpty was used.